### PR TITLE
V0.1.6

### PR DESCRIPTION
**Pull request by reiniervlinschoten**
Updates to move version 2022.4 of the Castor EDC API

### CODE REVIEW

File: castoredc_api/study/castor_objects/castor_data_point.py [hunk 1]
```diff
@@ -127,9 +127,14 @@ def __interpret_datetime(self, datetime_format: str):
...
                 new_value = "Missing value not recognized"
         else:
-            new_value = pd.Period(
-                datetime.strptime(self.raw_value, "%d-%m-%Y;%H:%M"), freq="S"
-            ).strftime(datetime_format)
+            try:
+                new_value = pd.Period(
+                    datetime.strptime(self.raw_value, "%d-%m-%Y;%H:%M"), freq="S"
+                ).strftime(datetime_format)
+            except:
```
Comment (IanButterworth) [inline castoredc_api/study/castor_objects/castor_data_point.py:134]:
```suggestion
            except ValueError:
```

Comment (reiniervlinschoten) [reply castoredc_api/study/castor_objects/castor_data_point.py:134]: Have changed this, thank you

**Comment by IanButterworth** [comment]
Friendly reminder to release this, to get fix #81 out. Thanks

**Review by reiniervlinschoten — COMMENTED** [review]
Okay for now to solve #81, still needs changes for upcoming name changes and rate limiting.

**Pull request merged**